From 64d881b955933200c843aa731c7a99412e87874a Mon Sep 17 00:00:00 2001
From: Prasanna Kumar M R <prasannax.k.mr@intel.com>
Date: Mon, 5 Aug 2013 17:53:06 +0530
Subject: init.rc updates for secure playback

Update init.rc to include the Media Server and DRM
Server in the intel_sec group so that they can access
MEI driver

Issue: APDEV-504

Category: device enablement
Domain: AOSP.Security
Origin: internal
Upstream-Candidate: no, Intel specific group

Change-Id: I5cbbb0078ec31ccfefb7321fcca3dcbef2579b1d
---
 rootdir/init.rc |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index c13f723..6fd56bc 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -492,12 +492,12 @@ service zygote /system/bin/app_process -Xzygote /system/bin --zygote --start-sys
 service drm /system/bin/drmserver
     class main
     user drm
-    group drm system inet drmrpc
+    group drm system inet drmrpc intel_sec
 
 service media /system/bin/mediaserver
     class main
     user media
-    group audio camera inet net_bt net_bt_admin net_bw_acct drmrpc mediadrm
+    group audio camera inet net_bt net_bt_admin net_bw_acct drmrpc mediadrm intel_sec
     ioprio rt 4
 
 service bootanim /system/bin/bootanimation
-- 
1.7.4.1

